plugins/ci/pkg/ci/ci.go

```diff
@@ -459,11 +459,15 @@ func ProcessHelmTemplates(configurationObject models.Configuration, configFolder
 		if err != nil {
 			return err
 		}
+		filefolder := configFolder
+		if strings.HasPrefix(filefolder, helmObject.Name) {
+			filefolder = strings.Replace(configFolder, helmObject.Name, helmObject.Path, 1)
```

Comment:
This will just template out the charts to a different directory (I think it might template them overtop where the chart is). The checking for prefix needs to happen in GetAllResources that way we can keep the `helmObject.Name` so the same chart could be processed multiple times for different values files.

Reply:
would this place here be the right place? 
https://github.com/FairwindsOps/insights-plugins/blob/main/plugins/ci/pkg/ci/ci.go#L61

Reply:
Oh yeah that looks perfect!